posthog/tasks/usage_report.py

```diff
@@ -375,6 +390,34 @@ def get_teams_with_recording_count_total() -> List[Tuple[int, int]]:
     return result
 
 
+def get_teams_with_hogql_metric(
+    begin: datetime,
+    end: datetime,
+    query_types: List[str],
+    access_method: str = "",
+    metric: Literal["read_bytes", "read_rows", "query_duration_ms"] = "read_bytes",
+) -> List[Tuple[int, int]]:
+    if metric not in ["read_bytes", "read_rows", "query_duration_ms"]:
+        # :TRICKY: Inlined into the query below.
+        raise ValueError(f"Invalid metric {metric}")
+    result = sync_execute(
+        f"""
+        WITH JSONExtractInt(log_comment, 'team_id') as team_id,
+             JSONExtractString(log_comment, 'query_type') as query_type,
+             JSONExtractString(log_comment, 'access_method') as access_method
+        SELECT team_id, sum({metric}) as count
+        FROM system.query_log
+        WHERE (type = 'QueryFinish' OR type = 'ExceptionWhileProcessing')
```

Comment:
you should have `is_initial_query` here, else we'll double count due to distributed queries